### PR TITLE
Add Classes

### PR DESCRIPTION
**Pull request by jadibdev**
### Description
Created the Person, Student, Teacher classes, and declared their variables & methods.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
+  end
```
Comment (brytebee) [inline person.rb]: - [x] **Required:** This method should be placed above the `private` keyword. It is required to be a `public method`. The `of_age?` method would still be able to access it.

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super
```
Comment (brytebee) [inline student.rb]: - [x] **Required:** Update the `initialize` and `super` as below to be able to inherit properly from the parent class.

```
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```
Comment (brytebee) [inline teacher.rb]: - [x] **Required:** Refactor this method as above.

File: README.md [hunk 4]
```diff
@@ -1 +1,31 @@
-# school__library
\ No newline at end of file
+## Description
```
Comment (brytebee) [inline README.md:5]: - [x] **Required:** kindly add a title to your `Readme`. It is professional for your documentation to be properly titled.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @jadibdev👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design and clean Ruby.✔️
- App functions as it should.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 2 above)
Comment (jadibdev) [reply student.rb]: @brytebee Thanks for your feedback, during my reading I saw that calling super without arguments would automatically inherit age, name, and parent_permission ... is that not the case?

**Review by dasileker — APPROVED** [review]
### **Hi @jadibdev,** 

### **Nice job on making the change requested by CR**

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**